frontend/public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>musiCompass</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
nice 🤣 no JS no app

Reply:
line 27 changes the app name in the browser tab. line 30 is not shown as long as everything is running fine. constructive input only, please